packages/php-wasm/universal/src/lib/urls.ts

```diff
@@ -59,3 +61,36 @@ export function ensurePathPrefix(path: string, prefix: string): string {
 	}
 	return prefix + path;
 }
+
+/**
+ * Returns a absolute URL for the provided URL.
+ *
+ * If the provided URL is relative, it will be prepended with the base URL.
+ * If a absolute URL is provided, it will return the provided URL and ignore
+ * the base URL.
+ *
+ * @param  url     - The URL to convert to an absolute URL.
+ * @param  baseUrl - The base URL to use to convert the relative URL to an absolute URL.
+ * @returns The absolute URL.
+ */
+export function toAbsoluteUrl(url: string, baseUrl: URL): string {
+	if (url.startsWith('http')) {
```

Comment:
This would deem `http-or-https-which-one-should-you-choose.html` as an absolute URL